README.md

```diff
@@ -1,35 +1,59 @@
 
-## CyberArk Ansible Conjur Collection
+# CyberArk Ansible Conjur Collection
 
-****
+This collection contains plugins to be used for CyberArk Conjur & DAP (Dynamic Access Provider) hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
 
-### cyberark.conjur
+## Table of Contents
+- [CyberArk Ansible Conjur Collection](#cyberark-ansible-conjur-collection)
+  * [Requirements](#requirements)
+  * [conjur_variable Lookup Plugin](#conjur_variable-lookup-plugin)
+    + [Role Variables](#role-variables)
+    + [Dependencies](#dependencies)
+    + [Example Playbook](#example-playbook)
+  * [Conjur Ansible Role](#conjur-ansible-role)
+    + [Usage](#usage)
+    + [Role Variables](#role-variables-1)
+    + [Dependencies](#dependencies)
+    + [Example Playbook](#example-playbook-1)
+    + [Summon & Service Managers](#summon---service-managers)
+    + [Recommendations](#recommendations)
+  * [Contributing](#contributing)
+  * [License](#license)
 
-This collection contains plugins to be used for CyberArk Conjur & DAP hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
+<!-- Table of contents generated with markdown-toc
+http://ecotrust-canada.github.io/markdown-toc/ -->
 
-### Requirements
+## Requirements
 
-- [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or
-- [CyberArk Dynamic Access Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm) v10.x+
-- Ansible v2.9+
+- conjur_variable Lookup Plugin
+  - [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or
+  - [CyberArk Dynamic Access Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm) v10.x+
+  - Ansible >= 2.9
 
-### Role Variables
-
-None.
-<br>
-<br>
+- Conjur Role
+  - A running Conjur service that is accessible from the target nodes.
+  - Ansible >= 2.3.0.0
 
-## Plugins
-
-### conjur_variable Lookup Plugin
+## Installation 
+From terminal, run the following command:
+```sh
+ansible-galaxy collection install cyberark.conjur
+```
 
+## conjur_variable Lookup Plugin
 Fetch credentials from CyberArk Conjur using the controlling host's Conjur identity or environment variables.
 
 - The controlling host running Ansible has a Conjur identity. [More Information here](https://docs.conjur.org/latest/en/Content/Get%20Started/key_concepts/machine_identity.html) and here in [Conjur Ansible role project](https://github.com/cyberark/ansible-conjur-host-identity/)
-- Environment variables could be CONJUR_ACCOUNT, CONJUR_APPLIANCE_URL, CONJUR_CERT_FILE, CONJUR_AUTHN_LOGIN, CONJUR_AUTHN_API_KEY, CONJUR_AUTHN_TOKEN_FILE
 
-#### Example Playbook
+- Environment variables could be `CONJUR_ACCOUNT`, `CONJUR_APPLIANCE_URL`, `CONJUR_CERT_FILE`, `CONJUR_AUTHN_LOGIN`, `CONJUR_AUTHN_API_KEY`, `CONJUR_AUTHN_TOKEN_FILE`
```

Comment:
this could be improved, but it's fine to leave that as a follow-up since it's not directly related to this PR

can we file an issue for it, though?

Reply:
Definitely